### DISCUSSION TITLE
Potential Memory Leak

### DISCUSSION THREAD

**Discussion by taylor-stein** [umami-software/umami#1488] [General]
Running v1.36.1, deployed via Railway, and I see what appears to be some sort of potential memory leak. Noticed this as my server is constantly restarting after hitting a memory limit. Is anyone else experiencing this? 

![image](https://user-images.githubusercontent.com/59148531/182228176-0f457386-ea42-4e4a-a64f-1e681c34b193.png)

**Comment by JokerQyou** [comment]
`v1.36.1` does seem to use more memory than previous versions (I switched from `v1.32` though, so maybe it's due to Prisma introduced), but I don't see memory leak in my docker installation.

![image](https://user-images.githubusercontent.com/1465267/182274467-91fd297a-cb26-4c89-a4fe-8816b6041c79.png)

**Comment by taylor-stein** [comment]
The only unique aspect of my usage of Umami is that I'm working around being able to send custom event payloads by modifying the `event_value` to capture the relevant data I need and parse it out down stream. Could these be potentially stored in memory, causing the memory leak over time?
